### PR TITLE
Mr Nanotrasen, a new admeme spawn

### PR DESCRIPTION
**Pull request by noctyrnal**
## About the PR

Based of emisse's insane lore drop, admins can now spawn Mr Nanotrasen, a space dragon with a new sprite, a PDA and Centcomm certified plated teeth. When he is killed, he can be butchered to drop 300K Spesos (Nanotrasen's profit)

## Why / Balance

Funny PR

## Technical details

N/A

## Media

- [ X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

f54daf5aede5a8c90a05c5464d53f58c

## Breaking changes

N/A

**Changelog**

:cl:
- add: Adds a new admeme spawn, Mr Nanotrasen the space dragon.

### CODE REVIEW

**Comment by UbaserB** [comment]
Why is he not white

**Comment by noctyrnal** [comment]
> Why is he not white

he can be any colour :)

**Comment by UbaserB** [comment]
> > Why is he not white
> 
> he can be any colour :)

he needs to be white specifically because thats whats stated in the lore

**Comment by deltanedas** [comment]
racism in the ss14 /!\\

**Comment by UbaserB** [comment]
> racism in the ss14 /!\

https://discord.com/channels/310555209753690112/675078881425752124/1222101235327565864

**Review by Flareguy — COMMENTED** [review]
The tie should really probably not be on the unshaded layer, it looks pretty weird here.

**Comment by noctyrnal** [comment]
> The tie should really probably not be on the unshaded layer, it looks pretty weird here.

issue is is that if i put it on the shaded layer then it fucks with the random colouring of the dragon, making the tie look bad. I can remove the random colouring if the unshaded tie is too much of a putoff though.

**Comment by Flareguy** [comment]
> > The tie should really probably not be on the unshaded layer, it looks pretty weird here.
> 
> issue is is that if i put it on the shaded layer then it fucks with the random colouring of the dragon, making the tie look bad. I can remove the random colouring if the unshaded tie is too much of a putoff though.

You can just make another layer that doesn't get colored, no?
You should probably remove the random coloring anyways, though, since he's supposed to be white.

**Comment by noctyrnal** [comment]
> > > The tie should really probably not be on the unshaded layer, it looks pretty weird here.
> > 
> > 
> > issue is is that if i put it on the shaded layer then it fucks with the random colouring of the dragon, making the tie look bad. I can remove the random colouring if the unshaded tie is too much of a putoff though.
> 
> You can just make another layer that doesn't get colored, no? You should probably remove the random coloring anyways, though, since he's supposed to be white.

ill just take the colouring away, saves me work.

**Comment by lzk228** [comment]
> > > Why is he not white
> > 
> > 
> > he can be any colour :)
> 
> he needs to be white specifically because thats whats stated in the lore

lmao, since when game depends on lore?

**Comment by UbaserB** [comment]
> > > > Why is he not white
> > > 
> > > 
> > > he can be any colour :)
> > 
> > 
> > he needs to be white specifically because thats whats stated in the lore
> 
> lmao, since when game depends on lore?

since they said it was lore :clueless:

**Comment by Moomoobeef** [comment]
NT only makes 300k?

**Comment by noctyrnal** [comment]
> NT only makes 300k?

they gambled the rest of it away to get thronglers

File: Resources/Prototypes/InventoryTemplates/mr_nanotrasen_inventory_template.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  slots:
+    - name: id
+      slotTexture: id
+      slotFlags: IDCARD
+      slotGroup: SecondHotbar
+      stripTime: 6
+      uiWindowPos: 2,1
+      strippingWindowPos: 2,4
+      dependsOn: jumpsuit
+      displayName: ID
```
Comment (metalgearsloth) [inline Resources/Prototypes/InventoryTemplates/mr_nanotrasen_inventory_template.yml:12]: Why's it in its own yml + please follow yaml styleguide for naming prototypes.

File: Resources/Prototypes/Entities/Mobs/Player/dragon.yml [hunk 2]
```diff
@@ -164,6 +164,64 @@
...
+        alive: Rainbow
+  - type: Sprite
+    sprite: Mobs/Aliens/Carps/mr_nanotrasen.rsi
+    noRot: true
+    layers:
+    - map: [ "enum.DamageStateVisualLayers.Base" ]
+      state: alive
+    - map: [ "enum.DamageStateVisualLayers.BaseUnshaded" ]
+      state: alive-unshaded
+      shader: unshaded
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Mobs/Player/dragon.yml:190]: Supposed to be white.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
Don't copy-paste existing sprites for admemes.

**Pull request closed without merging**